django/contrib/sites/__init__.py

```diff
@@ -1 +1,20 @@
+from django.apps import apps as django_apps
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+
+
+def get_site_model():
+    """
+    Returns the Site model that is active in this project.
+    """
+    try:
```

Comment:
Docstring should be `Return` not `Returns`, see https://www.python.org/dev/peps/pep-0257/#one-line-docstrings